lighthouse-core/test/computed/metrics/timing-summary-test.js

```diff
@@ -7,8 +7,8 @@
 
 const TimingSummary = require('../../../computed/metrics/timing-summary.js');
 
-const trace = require('../../fixtures/traces/frame-metrics-m89.json');
-const devtoolsLog = require('../../fixtures/traces/frame-metrics-m89.devtools.log.json');
+const trace = require('../../fixtures/traces/frame-metrics-m90.json');
+const devtoolsLog = require('../../fixtures/traces/frame-metrics-m90.devtools.log.json');
```

Comment:
I tried to avoid changing our expectations for LCP-AF and FCP-AF in any tests that I could. This is the only place I couldn't.